src/products.js

```diff
@@ -0,0 +1,117 @@
+const { log, saveFiles } = require('cozy-konnector-libs')
+const { rootUrl, request } = require('./request')
+const cozyhelpers = require('./cozyhelpers')
+
+const tableUrl = rootUrl + '/webapp/wcs/stores/controller/ec/products/table'
+const billPath = '/webapp/wcs/stores/controller/OrderBillDisplay'
+const firstPageNum = 1
+
+module.exports = {
+  fetchBills
+}
+
+function fetchBills(folderPath) {
+  return fetchPagesCount()
```

Comment:
:100: You could have done some micro-optimisation by grabbing the count from the first page without re-dwnloading it but this is way more readable! Awesome!

Reply:
I had exactly the same hesitation on my previous connector 😄 
I could improve it but I'm always uncomfortable because connector testing is quite tedious for the moment.